assets/scss/_includes/_grid.scss

```diff
@@ -16,8 +16,28 @@
 				width: calc( 100% / $cols );
 			}
 			$cols: $cols + 1;
+		
 		}
-
+	
 	}
 
 }
+
+.llms-flex-cols {
+	display: flex;
+	flex-flow: row wrap;
+
+	[class*="llms-col"] {
+   		flex: 0 1 auto;
```

Comment:
This is the initial `flex` value, so we could omit it here, although I'm more inclined to add it as you did, just to be a little bit more sure the cols are not inheriting a different `flex` value from the themes.